### ISSUE TITLE
Links not able to be easily opened

### ISSUE THREAD

**Issue by Uprise42** [hariettem/Artemis#370]
**Describe the bug**
Clicking the link address itself doesn't open the link. 

**To Reproduce**
1. Go to any feed
2. Click on the link address 
3. Nothing happens

**Expected behavior**
The link should happen

**Screenshot**
https://github.com/hariettem/Artemis/assets/58179605/32c50d13-1902-4110-9828-c4ceb4762923

**Smartphone (please complete the following information):**
 - Device: iPhone 14 Pro
 - OS: iOS17
 - Version: 0.2.0

Update: 2308.07.23.44
(7f74bc59-35eb-403d-a7fd-44d052f735cc)}

Channel: preview

**Additional context**
You can Force Touch the link to open in a separate window, or if there is a picture attached you can click the photo to open the link. But clicking the text does nothing.

**Comment by its-yourmom** [comment]
This is also happening on iPhone 11 Pro v 16.1.1
Version: 0.2.0

Update: 2308.08.21.12
(68af0e57-6ec5-4d60-8b0b-b15da6a24a59)}

Channel: preview

**Comment by RheingoldRiver** [comment]
To clarify, this is 100% reproduction right (happens every single time)?

**Comment by its-yourmom** [comment]
Yes, this happens when I touch the links. They first start by the words speeding by really fast. But it doesn’t open the link. So I have either touch and hold and sometimes I get to the link, or I touch the picture ( if there’s a picture attached that is) and then I get to the link.
